cfgov/v1/models/base.py

```diff
@@ -58,15 +53,6 @@ def shared_q(self):
     def shared(self):
```

Comment:
@chosak I started down that path, and it turned out to be a bit of a rabbit hole.  Should be able to clean it up, but I'd prefer to do that in v2 of clean-up.